x/bep3/spec/02_state.md

```diff
@@ -0,0 +1,82 @@
+# State
+
+## Parameters and genesis state
+
+`Paramaters` define the rules according to which swaps are executed. Parameter updates can be made via on-chain parameter update proposals.
+
+```go
+// Params governance parameters for bep3 module
+type Params struct {
+	BnbDeputyAddress sdk.AccAddress `json:"bnb_deputy_address" yaml:"bnb_deputy_address"`
+	MinBlockLock     int64          `json:"min_block_lock" yaml:"min_block_lock"`
```

Comment:
I would add a comment to block-lock params that they represent block height span, not time or some other measure.

Reply:
https://github.com/Kava-Labs/kava/pull/450/commits/2516b8d15cef7d1fa013aaef9ddb75171e7ad413